_sass/components/_navigation.scss

```diff
@@ -0,0 +1,61 @@
+.c-navigation {
+  position: absolute;
+  width: 100%;
+
+  .c-navigation__logo {
+    width: 4rem;
+    @media (min-width:$bp-sm){
+    width:6rem;
+  }
```

Comment:
Looks like the indentation is a little off here, when we pair I can show you a really cool tool that I was shown as an apprentice that will automatically fix indentation in files